src/nl/codefox/contextvh/tygronGoals.mod2g

```diff
@@ -21,17 +21,32 @@ module tygronGoals {
 		then adopt(doneBuying(MultiPoly)).
 	
 	% Example of a building we will demolish
```

Comment:
I agree, I believe it is best to remove all these examples in another branch after all current branches have been merged.